abci/example/counter/counter.go

```diff
@@ -0,0 +1,85 @@
+package counter
```

Comment:
Wasn't this file supposed to have been removed in https://github.com/tendermint/tendermint/commit/ea271c534a6955b8abc9e2b5b62bd16a437a81a4?